packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -80,6 +83,9 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
     };
 
     this._droppingClassNames = '';
+
+    this._checkboxId = getId('checkbox');
```

Comment:
Is it possible to compute these deterministically somehow by using a unique ID assigned to the whole list and then appending the row id? Doing that would keep test snapshot churn down!

Reply:
Basically, have `DetailsList` pass a `rowId` to each row in `onRenderRow`, which is computes in advance from its own unique ID and the `itemIndex`. Then have `DetailsRowFields` and `onRenderCheck` simply compute the expected id of the header element.

Reply:
Gotcha - will try to implement these changes!